.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": ["env"],
+  "plugins": ["transform-object-rest-spread"]
+ }
+ 
```

Comment:
A line break is needed here.

Reply:
Thanks, for pointing that out, fixed now.